src/main/java/com/watch/shop/app/model/WatchStore.java

```diff
@@ -0,0 +1,86 @@
+package com.watch.shop.app.model;
+
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.Setter;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+public class WatchStore {
```

Comment:
model in MVC stands for Service, Repository , Entities, it's logic term, so it's ok to put entities into Model package and have Service as other package, it's acceptable, but as classic structure:

controller
model 
  entity package (Color, Watch, WatchType)
  WatchService
  WatchServiceImpl
  WatchStore
  
  or put WatchStore to model/repository package